3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/AnonIpEnrichment.scala

```diff
@@ -52,23 +55,30 @@ object AnonIpEnrichment extends ParseableEnrichment {
    *        Must be a supported SchemaKey for this enrichment
    * @return a configured AnonIpEnrichment instance
    */
-  def parse(config: JValue, schemaKey: SchemaKey): ValidatedNelMessage[AnonIpEnrichment] =
-    isParseable(config, schemaKey).flatMap(conf => {
+  def parse(config: JValue, schemaKey: SchemaKey): ValidatedNelMessage[AnonIpEnrichment] = {
+
+    def extractParam(name: String) =
+      ScalazJson4sUtils.extract[Int](config, "parameters", name)
+
+    isParseable(config, schemaKey).flatMap(_ => {
       (for {
-        param  <- ScalazJson4sUtils.extract[Int](config, "parameters", "anonOctets")
-        octets <- AnonOctets.fromInt(param)
-        enrich = AnonIpEnrichment(octets)
+        paramIPv4Octet   <- extractParam("anonOctets")
+        paramIPv6Segment <- extractParam("anonSegments").orElse(paramIPv4Octet.success)
```

Comment:
This `default` value for number of anonymized segments is provided to not break backwards compatibility with "patch" release, which R117 was meant to be.